.circleci/config.yml

```diff
@@ -43,6 +46,14 @@ jobs:
             docker run --volumes-from mms_test_configs --network container:mms -t postman/newman run artifacts.postman_collection.json -e test-env.json --delay-request 500
             docker run --volumes-from mms_test_configs --network container:mms -t postman/newman run elastic.postman_collection.json -e test-env.json --delay-request 500
             docker run --volumes-from mms_test_configs --network container:mms -t postman/newman run groups.postman_collection.json -e test-env.json --delay-request 500
+            
+            docker stop mms
+            mkdir -p jacocoOutput
+            docker cp mms:/mms/jacocoOutput/jacoco-it.exec ./jacocoOutput/jacoco-it.exec
+            cp -r ./*/build/classes/* TEMPORARYclassfiles/
```

Comment:
because the class files aren't actually built here, you want to run the cli report in the scan job, after the the run step - instead of checkout in scan, you can attach workspace (like the deploy_snasphot job) 
also you want to get only the build/classes/java/main so it doesn't count test files

Reply:
add requires: build_and_test to scan in the workflows section so scan comes after build_and_test

Reply:
That makes sense. Thank you!